tests/DocumentTest.php

```diff
@@ -425,4 +425,37 @@ public function testFromFile()
         $this->assertInstanceOf('Sokil\Vast\Document', $document::fromFile(__DIR__ . '/vast.xml'));
     }
 
+    /**
+     * VPAID creative test
+     */
+    public function testVpaidCreative()
+    {
+        $factory = new Factory();
+        $document = $factory->create('3.0');
+
+        $ad = $document
+            ->createInLineAdSection()
+            ->setId('test-vpaid')
+            ->setAdSystem('Ad Server Name')
+            ->setAdTitle('VPAIDPreRoll')
+        ;
+        /** @var \Sokil\Vast\Creative\InLine\Linear $creative */
+        $creative = $ad->createLinearCreative();
+        $creative
+            ->setDuration(10)
+            ->createAdParameters()
```

Comment:
why you need setter to add node content? Maybe better jist to add array as argument to createAdParameters?